format/index.md

```diff
@@ -249,8 +250,8 @@ which **MUST** contain at least one of the following:
   * `related`: a member, whose value is a "related resource link", as defined
     below.
 * `data`: a member, whose value represents "resource linkage" (defined below).
-* `meta`: a member that contains non-standard meta-information about the
-  relationship.
+* `meta`: a member, whose value is a [meta] object that contains non-standard
```

Comment:
a [meta] object containing non-standard meta-information about the relationship.